torch/nn/modules/module.py

```diff
@@ -665,7 +665,7 @@ def to(self, *args, **kwargs):
                     "if a complex module does not work as expected.")
 
         def convert(t):
-            if convert_to_format is not None and t.dim() == 4:
+            if convert_to_format is not None and t.dim() in (4, 5):
```

Comment:
This may negatively affect other 5d module performance if they don't have an optimized `channels_last_3d` path and users convert the module to `channels_last_3d` (rare case?).